CONTRIBUTING.md

```diff
@@ -0,0 +1,67 @@
+# Contributing
+
+## Reporting Issues
+
+If you have found what you think is a bug, please [file an issue](https://github.com/pmndrs/zustand/issues/new).
+
+For usage questions, prefer [starting a discussion](https://github.com/pmndrs/zustand/discussions/new).
+
+## Suggesting new features
+
+If you are here to suggest a feature, first create an issue if it does not already exist. From there, we will discuss use-cases for the feature and then finally discuss how it could be implemented.
+
+## Documentation
+
+- Fork the [Zustand repo](https://github.com/pmndrs/zustand/) into your own Github account
+- Locally, clone down your fork
+- Separately, clone the [Pmdrs-Website](https://github.com/pmndrs/website}
+  - this runs most of the doc websites under the pmndrs banner, including react three fiber and (eventually) zustand
+  - Switch to the `docs`-branch.
+- Now, you should have two repositories locally
+- Inside the website folder, run `npm install` and then `npm run dev`
+  - This will launch the docs site locally, that you should be able to open in the browser and see the various doc sites
```

Comment:
```suggestion
  - This will launch the website locally. You should be able to open and see the various documentation sites
```

 - Some of my original notes in that discussion were shorthand so cleaning up a bit